run.py

```diff
@@ -872,6 +872,9 @@ def run_test(dirpath, args, cwd, suricata_config):
         elif results["success"] > 0:
             with lock:
                 count_dict["passed"] += 1
+        check_output = subprocess.call(["../suricata-verify/check-eve.py", outdir, "-q"])
```

Comment:
We need to use TOPDIR here to get the path correct.  I can suricata-verify like `../../suricata-verify.run.py` from my Suricata source directory, and the path is not found.